packages/content/src/enrichments.ts

```diff
@@ -262,55 +259,99 @@ export function _enrichDates(content: IHubContent): IHubContent {
   return newContent;
 }
 
-const fetchGroupIds = (
-  content: IHubContent,
-  requestOptions?: IHubRequestOptions
-): Promise<string[]> => {
-  return getItemGroups(content.id, requestOptions).then((response) => {
-    const { admin, member, other } = response;
-    return [...admin, ...member, ...other].map((group) => group.id);
-  });
+const enrichGroupIds = (
+  input: IPipeable<IHubContent>
+): Promise<IPipeable<IHubContent>> => {
+  const { data: content, stack, requestOptions } = input;
+  const opId = stack.start("enrichGroupIds");
```

Comment:
If these are ever going to be run on a list of content objects, i'd suggest using 
```js
...startOperation({ id: `enrichGroupIds-${data.id}`, type: 'enrichGroups', inputs: { id: data.id}})`
```
it's a little more verbose, but you'll be able to trace back to _which_ item was involved in a failure vs just knowing something failed... I'd do this in all of these enrichment fns

Reply:
It's odd the other tests didn't fail for me locally, but they are now.

Reply:
tests are passing. I think we should merge this as is. I'll be revisiting this code soon for #543 at which point I can do the more verbose `startOperation()` call.